week2/index.js

```diff
@@ -0,0 +1,157 @@
+"use-strict";
+
+const fs = require('fs');
+const FILENAMES = { help: "help.txt", todo: "todo.json" }
+
+let readFiles = (filename) => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filename, "utf8", (err, data) => {
+            if (err) {
+                if (err.code === "ENOENT") {
+                    console.log(`Sorry, this file doesn't exist, but new file has been created : ${filename}`);

```

Comment:
To clarify my comment earlier - READING from a non-existent file doesn't create that file. But WRITING to a non-existent file will create a new file.

Reply:
Also, you shouldn't write a log message that says "something HAS been done" before you've actually done it, because it's not true yet. On this line, you could say "Sorry, this file doesn't exist, but I will attempt to create it: ", and later on, after it's been done successfully, you can say that it "has been created".